.github/workflows/trigger-v1-tests.yml

```diff
@@ -0,0 +1,18 @@
+name: trigger v1 tests
+
+on:
+  schedule:
+    # At 3 AM UTC every day
+    - cron: 0 3 * * *
+
+jobs:
```

Comment:
On second thought, can you also trigger the canary release from v1, every night too?